src/internal.c

```diff
@@ -596,15 +567,18 @@ static int ExportKeyState(WOLFSSL* ssl, byte* exp, word32 len, byte ver)
     exp[idx++] = keys->encryptionOn;
     exp[idx++] = keys->decryptedCur;
 
-#ifdef WORD64_AVAILABLE
-    c64toa(keys->window, exp + idx);     idx += OPAQUE64_LEN;
-    c64toa(keys->prevWindow, exp + idx); idx += OPAQUE64_LEN;
-#else
-    c32toa(keys->window, exp + idx);     idx += OPAQUE32_LEN;
-    c32toa(0, exp + idx);                idx += OPAQUE32_LEN;
-    c32toa(keys->prevWindow, exp + idx); idx += OPAQUE32_LEN;
-    c32toa(0, exp + idx);                idx += OPAQUE32_LEN;
-#endif
+    c16toa(WOLFSSL_DTLS_WINDOW_WORDS, exp + idx); idx += OPAQUE16_LEN;
```

Comment:
I'll break it up into two sizes. They should never be different sizes, but they are two variable length items. And I'll use the sizes in octets. That seems to be the pattern.

I was using the array length.